.gitignore

```diff
@@ -16,4 +17,5 @@ npm-debug.log
 gen/
 .nyc_output/
 coverage/
-package-lock.json
+.rpt2_cache
```

Comment:
What is a `.rpt2_cache` file?

Reply:
This is TypeScript2 Rollup plugin's cache.